src/test/lumo/lumo/build_api_tests.cljs

```diff
@@ -337,37 +337,52 @@
         (build/build (build/inputs (path/join inputs "emit_node_requires_test/native_modules.cljs")) opts cenv))
       (is (fs/existsSync (path/join out "emit_node_requires_test/native_modules.js")))
       (is (true? (boolean (re-find #"emit_node_requires_test\.native_modules\.node\$module\$path\.isAbsolute"
-                            (slurp (path/join out "emit_node_requires_test/native_modules.js"))))))
+                                   (slurp (path/join out "emit_node_requires_test/native_modules.js"))))))
       (is (empty? @ws))))
   (test/delete-node-modules))
 
-(deftest cljs-test-compilation
-
-  (testing "success"
-    (let [out (path/join (test/tmp-dir) "compilation-test-out")
-          root "src/test/cljs_build"]
+#_(deftest cljs-test-compilation
```

Comment:
Why is this commented out?

Reply:
Sorry it is wip, I will put things back where they were